pkg/helpers/mapper.go

```diff
@@ -36,3 +35,28 @@ func OneToMany[valT GetMappingID[keyT], keyT comparable](keys []keyT, vals []val
 
 	return out
 }
+
+type GetMappingIDAndEmbedPtr[keyT comparable, embedPtr any] interface {
+	GetMappingID() keyT
+	GetEmbedPtr() embedPtr
+}
+
+func OneToManyEmbedded[valT GetMappingIDAndEmbedPtr[keyT, embedPtr], keyT comparable, embedPtr any](keys []keyT, vals []valT) [][]embedPtr {
```

Comment:
can you write some tests for this one?

Reply:
Added in [8257db6](https://github.com/CMS-Enterprise/easi-app/pull/2678/commits/8257db6732849f6017e54cc73204a53112fa9696)